.gitlab-ci.yml

```diff
@@ -1255,6 +1267,25 @@ latest_release:
     - inv -e docker.publish --signed-pull --signed-push datadog/agent:${RELEASE}-jmx datadog/agent:latest-jmx
     - inv -e docker.publish --signed-pull --signed-push datadog/dogstatsd:${RELEASE} datadog/dogstatsd:latest
 
+#
+# Use this step to delete a tag of a given image
+# - remove the leading dot from the name
```

Comment:
Uncommenting and commenting a job name like this seems fragile and error-prone. I understand we don't want to accidentally delete images, but is there a way to make this more usable?

Reply:
Totally agree that this is not the perfect way to enable/disable the job but I wanted to be consistant with the `latest_revert_to_previous_release` job here https://github.com/DataDog/datadog-agent/blob/d4b584b5d69220a3aa73ac2b0a83da07021fa6b2/.gitlab-ci.yml#L1251-L1268

Reply:
I see. Is this well-documented? Does this work for everyone else on the team? Is there a way to not run a job unless some conditions are set?

Reply:
will work on that, thanks 👍